controls/radchartview/populating-with-data/chartdatasource.md

```diff
@@ -225,12 +208,9 @@ Below you'll find a sample where this is demonstrated (the datasource is the sam
 							ItemsSource="{Binding ElementName=ChartDataSource1}">
 	    </local:MyBarSeries>
 	</telerik:RadCartesianChart>
-	{{endregion}}
-
-
-
-#### __C#__
+{{endregion}}
 
+#### __[C#] Example 5: Create custom series and overriding its aggregate function__

```

Comment:
Create custom series and override its aggregate function or  Creating custom series and overriding its aggregate function